docs/extend/extend-cli/cli-setting-up.md

```diff
@@ -19,8 +27,9 @@ Follow these steps:
 1. `cd` to your `zowe-tutorial` folder.
 2. `git clone https://github.com/zowe/zowe-cli-sample-plugin`
 3. `cd` to your `zowe-cli-sample-plugin` folder.
-4. `npm install`
-5. `npm run build`
+4. Switch your local branch to `lts-incremental`
```

Comment:
Since we already provide a `command-by-command` instruction, I suggest changing this statement to `git checkout lts-incremental`